specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-11-01/frontdoor.json

```diff
@@ -1,2350 +0,0 @@
-      "required": true,
-      "type": "string",
-      "x-new-pattern": "^[a-zA-Z0-9]+([-a-zA-Z0-9]?[a-zA-Z0-9])*$",
-      "pattern": "^[a-zA-Z0-9]+([-a-zA-Z0-9]?[a-zA-Z0-9])*$",
-      "minLength": 5,
-      "maxLength": 64,
-      "x-ms-parameter-location": "method",
-      "description": "Name of the Front Door which is globally unique."
-    },
-    "routingRuleNameParameter": {
-      "name": "routingRuleName",
-      "in": "path",
-      "required": true,
-      "type": "string",
-      "pattern": "^[a-zA-Z0-9]+(-*[a-zA-Z0-9])*$",
-      "x-previous-pattern": "^[-\\w\\d\\._]+$",
-      "minLength": 1,
-      "maxLength": 90,
-      "x-ms-parameter-location": "method",
-      "description": "Name of the Routing Rule which is unique within the Front Door."
-    },
-    "healthProbeSettingsNameParameter": {
-      "name": "healthProbeSettingsName",
-      "in": "path",
-      "required": true,
-      "type": "string",
-      "pattern": "^[a-zA-Z0-9]+(-*[a-zA-Z0-9])*$",
-      "x-previous-pattern": "^[-\\w\\d\\._]+$",
-      "minLength": 1,
-      "maxLength": 90,
-      "x-ms-parameter-location": "method",
-      "description": "Name of the health probe settings which is unique within the Front Door."
-    },
-    "loadBalancingSettingsNameParameter": {
-      "name": "loadBalancingSettingsName",
-      "in": "path",
-      "required": true,
-      "type": "string",
-      "pattern": "^[a-zA-Z0-9]+(-*[a-zA-Z0-9])*$",
-      "x-previous-pattern": "^[-\\w\\d\\._]+$",
-      "minLength": 1,
-      "maxLength": 90,
-      "x-ms-parameter-location": "method",
-      "description": "Name of the load balancing settings which is unique within the Front Door."
-    },
-    "backendPoolNameParameter": {
-      "name": "backendPoolName",
-      "in": "path",
-      "required": true,
-      "type": "string",
-      "pattern": "^[a-zA-Z0-9]+(-*[a-zA-Z0-9])*$",
-      "x-previous-pattern": "^[-\\w\\d\\._]+$",
-      "minLength": 1,
-      "maxLength": 90,
-      "x-ms-parameter-location": "method",
-      "description": "Name of the Backend Pool which is unique within the Front Door."
-    },
-    "frontendEndpointNameParameter": {
-      "name": "frontendEndpointName",
-      "in": "path",
-      "required": true,
-      "type": "string",
-      "pattern": "^[a-zA-Z0-9]+(-*[a-zA-Z0-9])*$",
-      "x-previous-pattern": "^[-\\w\\d\\._]+$",
-      "minLength": 1,
-      "maxLength": 255,
-      "x-ms-parameter-location": "method",
-      "description": "Name of the Frontend endpoint which is unique within the Front Door."
-    },
-    "rulesEngineNameParameter": {
-      "name": "rulesEngineName",
-      "in": "path",
-      "required": true,
-      "type": "string",
-      "pattern": "^[a-zA-Z0-9]+(-*[a-zA-Z0-9])*$",
-      "x-previous-pattern": "^[-\\w\\d\\._]+$",
-      "minLength": 1,
-      "maxLength": 90,
-      "x-ms-parameter-location": "method",
-      "description": "Name of the Rules Engine which is unique within the Front Door."
-    }
-  }
-}
```

Comment:
Why front door removed from this version?

Reply:
This new api changes are only related to web application firewall policy. So not including frontdoor.json